packages/knip/src/plugins/yarn/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { _glob } from '../../util/glob.js';
+
+// https://yarnpkg.com/features/constraints
+
+const title = 'Yarn';
+
+const enablers = 'This plugin is enabled when a `yarn.lock` file is found in the root folder.';
+
+const isEnabled: IsPluginEnabled = async ({ cwd }) => (await _glob({ cwd, patterns: ['yarn.lock'] })).length > 0;
```

Comment:
There's `_firstGlob` for this, any chance we could use that? Example: https://github.com/webpro-nl/knip/blob/main/packages/knip/src/plugins/github-actions/index.ts#L13-L14 (probably should move into a nice helper but we could consider this in another PR)

Reply:
Great! I've changed it 0b8a898